docs/UserGuide.md

```diff
@@ -296,6 +296,39 @@ Format: `priorityP INDEX priority/PRIORITY`
 Examples:
 * `listP` followed by `priorityP 2 priority/high` sets the 2nd project as high priority in TaskHub.
 
+
+### Mark a task as completed: `markT`
+
+Marks the specified task(s) of a specified project as completed in TaskHub.
+
+Format `markT pr/PROJECT_INDEX t/TASK_INDEX [MORE_TASK_INDEXES]`
+
+* Marks the tasks(s) at the specified `TASK_INDEX`(es), of the project at the specified `PROJECT_INDEX` as completed.
+* The project index refers to the index number shown in the displayed projects list.
+* The task index(es) refer to the index number(s) shown in the task list of the above specified project.
+* The index(es) **must be positive integers** 1, 2, 3,...
+
+Examples:
+* `listP` followed by `markT pr/1 t/1 3` marks the 1st and 3rd tasks of the 1st project as completed in TaskHub.
+* `findP Capstone` followed by `markT pr/2 t/2` marks the 2nd task in the 2nd project from the results of the `findP` command as completed.
```

Comment:
Perhaps it would be better to do `markT pr/1 t/2`, to make it more clear that it is the first project with the name `Capstone` that we are targetting? It would be unlikely (though not impossible) for `findP Capstone` to give 2 results in the filtered list. But I do understand that it is not within the realm of impossibility that there would be 2 results in the filtered list, therefore this is still a valid command and you may choose to not change it. Just thought that most users would not have similar project names and therefore they are more likely to do `pr/1` instead of `pr/2` or any other indexes.

Reply:
I see where you are coming from, but for now I think we can leave it be (I could add Ui screenshots later to accompany the description).

Reply:
I think it is fine to leave it as is because it's quite clear to the user that the indices used in the `markT` command would be based on the current shown list of `project`s and `tasks`.

Nonetheless, if the use case is to find a particular project and then remove tasks from that project, then Aslam's suggestion would be great.